src/character-sheet/CharacterSheet.js

```diff
@@ -0,0 +1,282 @@
+                    character.proficiencies.dexSavingThrow = false;
+                    character.proficiencies.conSavingThrow = false;
+                    character.proficiencies.intSavingThrow = false;
+                    character.proficiencies.wisSavingThrow = true;
+                    character.proficiencies.chaSavingThrow = true;
+                    break;
+                case "Cleric":
+                    character.proficiencies.strSavingThrow = false;
+                    character.proficiencies.dexSavingThrow = false;
+                    character.proficiencies.conSavingThrow = false;
+                    character.proficiencies.intSavingThrow = false;
+                    character.proficiencies.wisSavingThrow = true;
+                    character.proficiencies.chaSavingThrow = true;
+                    break;
+                case "Sorcerer":
+                    character.proficiencies.strSavingThrow = false;
+                    character.proficiencies.dexSavingThrow = false;
+                    character.proficiencies.conSavingThrow = false;
+                    character.proficiencies.intSavingThrow = true;
+                    character.proficiencies.wisSavingThrow = true;
+                    character.proficiencies.chaSavingThrow = false;
+                    break;
+                case "Pyromancer":
+                    character.proficiencies.strSavingThrow = false;
+                    character.proficiencies.dexSavingThrow = false;
+                    character.proficiencies.conSavingThrow = true;
+                    character.proficiencies.intSavingThrow = false;
+                    character.proficiencies.wisSavingThrow = false;
+                    character.proficiencies.chaSavingThrow = true;
+                    break;
+                case "Deprived":
+                    setDeprivedSavingThrows({
+                        deprivedStrSavingThrow: false,
+                        deprivedDexSavingThrow: false,
+                        deprivedConSavingThrow: false,
+                        deprivedIntSavingThrow: false,
+                        deprivedWisSavingThrow: false,
+                        deprivedChaSavingThrow: false,
+                    });
+                    setDeprivedSavingThrowsDialogOpen(true);
+                    break;
+                default:
+                    break;
+            }
+            setProficiencySelectionDialogOpen(true);
+        } else if (name === "enterCombat") {
+            character.inCombat = true;
+            setCombatPositionDialogOpen(true);
+        } else if (name === "exitCombat") {
+            character.inCombat = false;
+            character.position.combatPool = character.position.base;
+            if (character.position.current <= character.position.base) {
+                //do nothing
+            } else {
+                character.position.current = character.position.base;
+            }
```

Comment:
empty if blocks wouldnt be the best thing here, could we refactor this to have just the following structure
if(condition){
   statement
}